lib/adaptiveCardHelper.js

```diff
@@ -18,12 +18,13 @@ export function createCard(elements) {
         type: cardTypes.adaptiveCard,
         body: [],
         actions: [],
-        version: '1.0'
+        version: '1.2'
     };
     var body = toArray(elements);
     if (Array.isArray(elements) &&
         elements.length === 1 &&
-        isContainer(elements[0])) {
+        isContainer(elements[0]) &&
+        Object.keys(elements[0]).length === 2) {
```

Comment:
This check is kind of lengthy and from an outsider seems daunting, a comment here describing what is being checked could be useful.

Reply:
Or better yet, extract the checks to variables like `isSingleElement` `isElementContainer` etc along with the description.